### ISSUE TITLE
total-power-warn-threshold definition shall be uniform

### ISSUE THREAD

**Issue by arthurMll** [OpenNetworkingFoundation/TAPI#351]
I noted that the descripton and the structure of the total-power-warn-threshold-upper/lower leafs is contradictory. The description suggest that multiple values define the threshold (i.e., max,min, default...) however it is defined as a single leaf of type decimal64. 

```
            leaf total-power-warn-threshold-upper {
            	type decimal64 {
                	fraction-digits 7;
                }
                description "Configure the Max, Default and Min values for the Upper power threshold.";
            }
            leaf total-power-warn-threshold-lower {
            	type decimal64 {
                	fraction-digits 7;
                }
                description "Configure  Max, Default and Min values for lower power threshold.";
}
```

I suggest or modifying the description or change the leafs by containers and use the defined grouping:

```
        grouping total-power-threshold-pac {
            leaf total-power-upper-warn-threshold-default {
            	type decimal64 {
                	fraction-digits 7;
                }
                description "Can read the value of the default  threshold that was set";
            }
            leaf total-power-upper-warn-threshold-min {
            	type decimal64 {
                	fraction-digits 7;
                }
                description "Can read the value of the lower threshold that was set";
            }
            leaf total-power-upper-warn-threshold-max {
            	type decimal64 {
                	fraction-digits 7;
                }
                description "Can  read the value of the upper threshold that was set";
            }
            leaf total-power-lower-warn-threshold-default {
            	type decimal64 {
                	fraction-digits 7;
                }
                description "Can read the value of the default  threshold that was set";
            }
            leaf total-power-lower-warn-threshold-max {
            	type decimal64 {
                	fraction-digits 7;
                }
                description "Can  read the value of the upper threshold that was set";
            }
            leaf total-power-lower-warn-threshold-min {
            	type decimal64 {
                	fraction-digits 7;
                }
                description "Can read the value of the lower threshold that was set";
            }
            description "none";
}
```

**Comment by arthurMll** [comment]
Moreover, I think it is needed a clarification on whether the total-power-warn-threshold-upper/lower refers to the reception sensibility of the OTSi tapi-connectivity:end-point, either in the leaf name or in the description.

**Comment by bzeuner** [comment]
![greenshot](https://user-images.githubusercontent.com/6310277/46019425-878f2700-c0cb-11e8-99a6-5adc9f7a6ba9.png)

**Comment by karthik-sethuraman** [comment]
Fixed by PR #355
